abel/tools/center.py

```diff
@@ -276,10 +276,11 @@ def find_center_by_convolution(IM, **kwargs):
     len_conv = len(conv_0)/2
 
     #Take the first max, should there be several equal maxima.
-    center = (np.argmax(conv_0)/2, np.argmax(conv_1)/2)
+    # 10May16 - axes swapped - check this
```

Comment:
You are correct, I mislabelled the slices, although the returned tuple was correct. The `Lin_Basex` code deals with (x, y) coordinates, that are flipped for numpy arrays  (y, x) == (axis=0, 1).

Now correct:
```python
[etalon:.+PyAbel/examples]$ ipython3
      :
In [1]: import abel

In [2]: from scipy.ndimage import shift

In [3]: IM = abel.tools.analytical.sample_image()
In [4]: abel.tools.center.find_center(IM, center='com')
Out[4]: (180.00000000000006, 180.00000000000006)

In [5]: abel.tools.center.find_center(IM, center='convolution')
Out[5]: (180.0, 180.0)

In [6]: IMx = shift(IM, (0, 50))

In [7]: abel.tools.center.find_center(IMx, center='com')
Out[7]: (179.99999999999997, 222.87188570338546)

In [8]: abel.tools.center.find_center(IMx, center='convolution')
Out[8]: (180.0, 230.0)
```